packages/jazz-tools/src/react/tests/media/image.test.tsx

```diff
@@ -0,0 +1,519 @@
+// @vitest-environment happy-dom
+import { Account, FileStream, ImageDefinition } from "jazz-tools";
+import { describe, expect, it, vi } from "vitest";
+import { Image } from "../../media/image";
+import { createJazzTestAccount } from "../../testing";
+import { render, screen, waitFor } from "../testUtils";
+
+describe("Image", async () => {
+  const account = await createJazzTestAccount();
+
+  vi.spyOn(Account, "getMe").mockReturnValue(account);
+
+  describe("initial rendering", () => {
+    it("should render nothing if coValue is not found", async () => {
+      const { container } = render(
+        <Image imageId="co_zMTubMby3QiKDYnW9e2BEXW7Xaq" alt="test" />,
+        { account },
+      );
+      const img = container.querySelector("img");
+      expect(img).toBeNull();
+    });
+
+    it("should render an empty image if the image is not loaded yet", async () => {
+      const original = FileStream.create({ owner: account._owner });
+      original.start({ mimeType: "image/jpeg" });
+      // Don't end original, so it has no chunks
+
+      const im = ImageDefinition.create(
+        {
+          original,
+          originalSize: [100, 100],
+          progressive: false,
+        },
+        {
+          owner: account,
+        },
+      );
+
+      const { container } = render(<Image imageId={im.id} alt="test" />, {
+        account,
+      });
+
+      const img = container.querySelector("img");
+      expect(img).toBeDefined();
+      expect(img!.getAttribute("width")).toBe(null);
+      expect(img!.getAttribute("height")).toBe(null);
```

Comment:
Why should these be `null` 

The originalSize is available, so these attributes should be defined

Reply:
It might break the layout. If the user didn't ask for the original sizes explicitly, then we preserve the browser's behaviour without any attribute. That's why I introduced the literal "original" for width/height.